polkadot/runtime/parachains/src/on_demand/mod.rs

```diff
@@ -349,6 +418,18 @@ where
 		});
 	}
 
+	/// Adds credits to the specified account.
+	///
+	/// Parameters:
+	/// - `who`: Credit receiver.
+	/// - `amount`: The amount of new credits the account will receive.
+	pub fn credit_account(who: T::AccountId, amount: BalanceOf<T>) {
+		Credits::<T>::mutate(who, |maybe_credits| {
+			let credits = maybe_credits.saturating_add(amount);
```

Comment:
nit: the naming `maybe_credits` suggests it's an optional type, whereas it's not (but it can be potentially zero)
```suggestion
		Credits::<T>::mutate(who, |old_credits| {
			let credits = old_credits.saturating_add(amount);
```